pipeline_steps/common.groovy

```diff
@@ -188,7 +266,11 @@ def conditionalStage(Map args){
   stage(args.stage_name){
     if (env.STAGES.contains(args.stage_name)){
         print "Stage Start: ${args.stage_name}"
-        args.stage()
+        ansiColor('xterm') {
+          timestamps(){
+            args.stage()
+          } //timestamps
+        } //ansiColor
         print "Stage Complete: ${args.stage_name}"
```

Comment:
Why not put everything inside timestamps, including the logging/debugging messages?